### DISCUSSION TITLE
下载时无法兼容，

### DISCUSSION THREAD

**Discussion by drLiamz** [volatile-static/Chartero#100] [Q&A] [answered]
下载各个版本皆显示与zotero不兼容，恳请解答
<img width="334" alt="屏幕截图 2024-02-14 190011" src="https://github.com/volatile-static/Chartero/assets/160017828/f6fde19c-8600-4b35-99d7-1aef661a10eb">

**Comment by volatile-static** [comment]
#76 最新版不支持zotero6了

**Reply by drLiamz** [reply]
请问我需要下载哪个版本的插件呢，因为我下了好多版本都不行

**Reply by volatile-static** [reply, marked as answer]
https://github.com/volatile-static/Chartero/tree/js_overlay
2.0.0之前的版本

**Reply by drLiamz** [reply]
已解决，十分感谢
